CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Contributing
```

Comment:
@developit @marvinhagemeister 
I added this document.

Reply:
Excellent.